impl/src/main/java/com/okta/idx/sdk/impl/client/BaseIDXClient.java

```diff
@@ -466,6 +466,30 @@ public TokenResponse token(String url, String grantType, String interactionCode,
         return tokenResponse;
     }
 
+    @Override
+    public void revokeToken(String tokenType, String token) throws ProcessingException {
+
+        StringBuilder urlParameters = new StringBuilder();
+        urlParameters.append("client_id=").append(clientConfiguration.getClientId());
+        urlParameters.append("&client_secret=").append(clientConfiguration.getClientSecret());
```

Comment:
Is there a way to do this on Java client side apps? We don't have a client secret!

Reply:
I am not sure, maybe you can check this with Mike.. IIRC he has achieved this with iOS.

Reply:
This is a V1 endpoint, so we do it in OIDC as well. 

https://github.com/okta/okta-oidc-android/blob/363015cc58c57f7e662f4cf3a63e724665682dea/library/src/main/java/com/okta/oidc/net/request/RevokeTokenRequest.java#L38

Looks like client secret isn't required.

Reply:
Yes, will set it optionally.

Reply:
updated.